### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by binarymason**
Per issue #423

### CODE REVIEW

**Comment by lencioni** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ Option                                  | Description
...
+`env`                                   | Hash of environment variables the hook should be run with. This is intended to be used as a last resort when an executable a hook runs is configured only via an environment variable. Any pre-existing environment variables with the same names as ones defined in `env` will have their original values restored after the hook runs. **NOTE:** Currently, only strings are accepted values.  Boolean values will raise an error (Working on a fix!). **WARNING**: If you set the same environment variable for multiple hooks and you've enabled parallel hook runs, since the environment is shared across all threads you could accidentally have these separate hooks trample on each other. In this case, you should disable parallelization for the hook using the `parallelize` option.
```
Comment (sds) [inline README.md]: I would prefer to not have "Working on a fix!" in the text here. This section is busy enough as it is.

**Comment by binarymason** [comment]
updated

**Comment by sds** [comment]
Merged as 9f16e23ba0a8f6feed8e1b1d82325b9d8871b08f. Thanks!

**Pull request closed without merging**